docs/resources/source_load_generator.md

```diff
@@ -91,6 +92,24 @@ Optional:
 - `schema_name` (String) The expose_progress schema name. Defaults to `public`.
 
 
+<a id="nestedblock--key_value_options"></a>
+### Nested Schema for `key_value_options`
+
+Required:
+
+- `keys` (Number) The number of keys in the source.
```

Comment:
I know in the docs, this number must be divisible by PARTITIONS * BATCH SIZE. Does this error propagate when someone tries running the DDL and if not, wondering if we should document it here.

Source: https://materialize.com/docs/sql/create-source/load-generator/#key-value